mikrotik/resource_dhcp_lease_test.go

```diff
@@ -64,6 +69,7 @@ func TestAccMikrotikDhcpLease_updateAddress(t *testing.T) {
 					testAccDhcpLeaseExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "address", originalIpAddress),
 					resource.TestCheckResourceAttr(resourceName, "macaddress", updatedMacAddress),
+					resource.TestCheckResourceAttr(resourceName, "blocked", originalBlockAccess),
 				),
 			},
```

Comment:
I see you added the `testAccDhcpLeaseUpdatedBlockAccess` function but it isn't used. Let's add another `resource.TestStep` here that shows we can update the blocked attribute from false to true.

Reply:
Nice call.  Hopefully addressed in latest push